CMakeLists.txt

```
@@ -18,30 +18,35 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with libVersioningCompiler. If not, see <http://www.gnu.org/licenses/>
 
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.20)
 
-project(libVersioningCompiler)
+project(
+  libVersioningCompiler
+  VERSION 2.0
```

```suggestion
  VERSION 3.0
```
Have a look at `git tag`. There already is a `v2.0`
I propose to name it `3.0`.